modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -183,6 +184,10 @@ export interface SignTransactionOptions extends BaseSignTransactionOptions {
     };
   };
   prv: string;
+  userKeychain?: Keychain;
+  backupKeychain?: Keychain;
+  bitgoKeychain?: Keychain;
+  taprootRedeemIndex?: number;
```

Comment:
`Keychain` is a bit too constraining here, for instance in tests we sometimes only have string xprv/xpubs 

`keychains: [string, string, string]` (extended pubkey format) is more convenient and matches the `prv` argument